src/Core/FormatFactorySettings.h

```diff
@@ -887,7 +887,7 @@ Controls validation of UTF-8 sequences in JSON output formats, doesn't impact fo
 Disabled by default.
 )", 0) \
     DECLARE(Bool, output_format_json_pretty_print, true, R"(
-When enabled, values of complex data types like Tuple/Array/Map in JSON output format in 'data' section will be printed in pretty format.
+This setting determines the formatting of nested structures such as Tuples, Maps, and Arrays within the `data` array for improved readability.
```

Comment:
I think we should keep reference to JSON in the description. How about:

```suggestion
This setting determines how nested structures such as Tuples, Maps, and Arrays are displayed within the `data` array when using the JSON output format.

For example, instead of output:

```json
"data":
[
	{
		"tuple": {"a":1,"b":2,"c":3},
		"array": [1,2,3],
		"map": {"a":1,"b":2,"c":3}
	}

],
```

The output will be formatted as:

```json
"data":
[
	{
		"tuple": {
			"a": 1,
			"b": 2,
			"c": 3
		},
		"array": [
			1,
			2,
			3
		],
		"map": {
			"a": 1,
			"b": 2,
			"c": 3
		}
	}

],
```

```

Reply:
Sorry, the formatting displays weirdly in the suggestion above. Here is the suggested description:

This setting determines how nested structures such as Tuples, Maps, and Arrays are displayed within the `data` array when using the JSON output format.

For example, instead of output:

```json
"data":
[
	{
		"tuple": {"a":1,"b":2,"c":3},
		"array": [1,2,3],
		"map": {"a":1,"b":2,"c":3}
	}

],
```

The output will be formatted as:

```json
"data":
[
	{
		"tuple": {
			"a": 1,
			"b": 2,
			"c": 3
		},
		"array": [
			1,
			2,
			3
		],
		"map": {
			"a": 1,
			"b": 2,
			"c": 3
		}
	}

],
```